agents/package_update_steps.py

```diff
@@ -82,3 +82,42 @@ async def add_fusa_label(state, next_step, dry_run, gateway_tools):
               logger.warning(f"Failed to add FuSa label to GitLab MR: {e}")
 
       return next_step
+
+  @staticmethod
+  async def add_blocking_comment(state, next_step, dry_run, gateway_tools):
+      """Add a blocking comment to the MR to prevent merge until Jotnar members approve.
+
+      Args:
+          state: The state of the workflow.
+          next_step: The next step to run.
+          dry_run: Whether to run the workflow in dry-run mode.
+          gateway_tools: The gateway tools to use.
+
+      Returns:
+          The next step to run.
+      """
+      if dry_run:
+          return next_step
```

Comment:
I think this shouldn't be here, but I understand it's consistent with the `add_fusa_label` method. I have more reservations about this "class" but they are out of scope of this PR.